recipes/r-ngsplot/build.sh

```diff
@@ -2,10 +2,17 @@
 outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
 mkdir -p $outdir
 mkdir -p $PREFIX/bin
-cp -R * $outdir 
+cp -R {bin,database,example,galaxy,lib,LICENSE} ${outdir}/
 #Set up links for 
-for f in $outdir/*; do
-    ln -s $outdir/* $PREFIX/bin
+for f in ${outdir}/bin/*; do
+    ln -s ${f} ${PREFIX}/bin
     fbname=$(basename "$f")
-    chmod 0755 ${PREFIX}/bin/$fbname
-done
\ No newline at end of file
+done
+
+#Activate/Deactivate dir
+ACTIVATE_DIR=$PREFIX/etc/conda/activate.d
+DEACTIVATE_DIR=$PREFIX/etc/conda/deactivate.d
+mkdir -p $ACTIVATE_DIR
+mkdir -p $DEACTIVATE_DIR
+cp $RECIPE_DIR/scripts/activate.sh $ACTIVATE_DIR/ngsplot-activate.sh
```

Comment:
In regards to the comment below for `scripts/activate.sh`:
Replace
```sh
cp $RECIPE_DIR/scripts/activate.sh $ACTIVATE_DIR/ngsplot-activate.sh
```
by something like
```sh
echo > "${ACTIVATE_DIR}/${PKG_NAME}-activate.sh" <<EOF
#!/bin/bash
export NGSPLOT="${outdir}"
export _CONDA_SET_NGSPLOT_ENV=1
EOF
```
. `${outdir}` will then have been evaluated at build time (becaus of using `<<EOF` with `EOF` being unquoted) and replaced by the correct output path (which `conda` will in turn adjust to the environment's path at install time).